### PR TITLE
Style reports layout

### PR DESCRIPTION
**Pull request by lmichalevich**
## Status

- [x] Ready
- [ ] In Progress

## Related Issues

https://github.com/demisto/etc/issues/38832

## Description
Styling table section & empty sections

## Snapshots (before)
Screen Shot 2021-09-12 at 18 43 22

## Snapshots (after)
Screen Shot 2021-09-13 at 14 28 19

### CODE REVIEW

File: src/components/Layouts/ReportLayout.less [hunk 1]
```diff
@@ -112,14 +114,15 @@ h1 {
...
-      .section-table,
+      .section-table:first-child,
       .section-markdown,
       .section-duration,
       .section-grouped-list,
       .global-section {
         &:not(.page-break-section) {
           border: 1px solid #DADBDB;
           padding: 10px 12px;
+          min-height: 140px;
```
Comment (yanyo87) [inline src/components/Layouts/ReportLayout.less:125]: from where is this value coming from

File: src/components/Sections/SectionTable.less [hunk 2]
```diff
@@ -2,18 +2,15 @@
   overflow-x: hidden;
   margin-top: 0 !important;
 
-  thead {
-    th {
-      overflow: hidden;
-      text-overflow: ellipsis;
-    }
-  }
-
```
Comment (yanyo87) [inline src/components/Sections/SectionTable.less:11]: please revert this change we want to show the table headers

File: src/components/Sections/SectionTable.less [hunk 3]
```diff
@@ -2,18 +2,15 @@
...
   th, td {
+    padding: 3px !important;
     border-top: 1px solid #DADBDB !important;
     border-left: 1px solid #DADBDB !important;
     border-bottom: none !important;
     border-right: none !important;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    width: 80px;
```
Comment (yanyo87) [inline src/components/Sections/SectionTable.less:13]: why we added a fixed width ?

**Review by yanyo87 — CHANGES_REQUESTED** [review]
please add screenshots for before and after 

why did you decide to do all this changes? 

do we have a bug or something ?

File: src/components/Sections/SectionTable.less (hunk 3 above)
Comment (lmichalevich) [reply src/components/Sections/SectionTable.less:13]: This acts like a minimum width (if there are a lot of items it will display only the first ones that fits)

File: src/components/Layouts/ReportLayout.less (hunk 1 above)
Comment (lmichalevich) [reply src/components/Layouts/ReportLayout.less:125]: https://www.figma.com/proto/o1YarvORod1pfGvejrw2hk/objects-reports?page-id=156%3A12956&node-id=156%3A13059&viewport=643%2C588%2C0.08749071508646011&scaling=min-zoom&starting-point-node-id=173%3A13959

File: src/components/Sections/SectionTable.less (hunk 2 above)
Comment (lmichalevich) [reply src/components/Sections/SectionTable.less:11]: I restored the code the hides the table headers:
```
          table {
            display: none;
          }
```

**Comment by lmichalevich** [comment]
@yanyo87 I updated the snapshots.

I think we have some kind of bugs with how we showed the indicators table.

In the main issue: ```In case of horizontal scroll, print only the columns that are visible in the web (when scroller is in the most left position)``` - this is why I added the width to the table columns

File: src/components/Sections/SectionTable.less [hunk 4]
```diff
@@ -10,6 +11,7 @@
   }
 
   th, td {
+    padding: 3px !important;
```
Comment (yanyo87) [inline src/components/Sections/SectionTable.less:14]: why do we need this ?

File: src/components/Sections/SectionTable.less [hunk 5]
```diff
@@ -1,6 +1,7 @@
 .section-table {
   overflow-x: hidden;
   margin-top: 0 !important;
+  height: 100%;
```
Comment (yanyo87) [inline src/components/Sections/SectionTable.less:4]: let's talk about this

File: src/components/Layouts/ReportLayout.less [hunk 6]
```diff
@@ -128,6 +122,7 @@ h1 {
         .section-table {
           border: none;
           padding: 0;
+          min-height: 0;
```
Comment (yanyo87) [inline src/components/Layouts/ReportLayout.less:125]: let's talk about this as well

File: src/components/Layouts/ReportLayout.less [hunk 7]
```diff
@@ -95,11 +94,6 @@ h1 {
             }
           }
         }
-        &.section-show-empty-state {
-          .section-table {
-            height: inherit !important;
-          }
-        }
       }
```
Comment (yanyo87) [inline src/components/Layouts/ReportLayout.less:97]: why did we removed this ?

File: src/components/Sections/SectionTable.less (hunk 4 above)
Comment (lmichalevich) [reply src/components/Sections/SectionTable.less:14]: override the default padding that the ui library adds there (it was larger and we need more space to show the content better)

File: src/components/Layouts/ReportLayout.less (hunk 7 above)
Comment (lmichalevich) [reply src/components/Layouts/ReportLayout.less:97]: It is not needed. you can see from the pictures that table without items looks well (the border needs to be until the end of the section)

File: src/components/Layouts/ReportLayout.less (hunk 6 above)
Comment (lmichalevich) [reply src/components/Layouts/ReportLayout.less:125]: Will be removed

**Review by yanyo87 — APPROVED** [review]
